partiql-ast/src/ast.rs

```diff
@@ -690,6 +697,27 @@ pub struct PathExpr {
     pub index: Box<Expr>,
 }
 
+#[derive(Visit, Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct ExcludePath {
+    pub root: AstNode<VarRef>,
+    pub steps: Vec<ExcludePathStep>,
+}
+
+/// A "step" within a path expression; that is the components of the expression following the root.
```

Comment:
Should clarify that this is a "step" within an `EXCLUDE` path

```suggestion
/// A "step" within an exclude path; that is the components of the exclude path following the root.
```